src/component/contents/DraftEditorBlock.react.tsx

```diff
@@ -276,7 +276,7 @@ export default class DraftEditorBlock extends React.Component<Props> {
       const leaves = leavesForLeafSet.map((leaf, jj) => {
         const offsetKey = DraftOffsetKey.encode(blockKey, ii, jj);
         const {start, end} = leaf;
-        const key = `${blockKey}-${start}-${end}`;
+        const key = `${blockKey}-${start}`;
```

Comment:
maybe add a comment about this change?

Reply:
Yeah, this is something we had actually already changed in our fork. But will add a comment.